app/src/components/map/MapFeatureList.tsx

```diff
@@ -3,24 +3,34 @@ import Checkbox from '@mui/material/Checkbox';
 import FormGroup from '@mui/material/FormGroup';
 import FormControlLabel from '@mui/material/FormControlLabel';
 import React from 'react';
+import IconButton from '@mui/material/IconButton';
+import DeleteForeverOutlinedIcon from '@mui/icons-material/DeleteForeverOutlined';
+import Box from '@mui/material/Box';
+import get from 'lodash-es/get';
+import { recalculateFeatureIds } from 'utils/mapBoundaryUploadHelpers';
 
 export interface MapFeatureListProps {
   features?: any;
   mask?: any; // Store what mask just changed
   maskState?: any; // Store which features are masked
   activeFeatureState?: any; // Store which feature is active
+  formikProps: any;
```

Comment:
Not really needed to pass formik as a prop. As you can construct another instance within the file.
`const formikProps = useFormikContext<IProjectLocationForm>();`

Reply:
Yeah... that would make it simpler.

Reply:
Done